OrbitService/CaptureServiceImpl.cpp

```diff
@@ -146,13 +150,37 @@ grpc::Status CaptureServiceImpl::Capture(
   }
   LOG("Client finished writing on Capture's gRPC stream: stopping capture");
 
-  tracing_handler.Stop();
-  LOG("LinuxTracingHandler stopped: perf_event_open tracing is done");
+  {
```

Comment:
Is there a reason to put it in its own scope?

Reply:
I'd actually either add comment explaining what goes on here, or move that to a separate method.

Reply:
The scope is just to limit the scope of `stop_threads`. I can see how it feels like this scope should be a separate function, although in my opinion it makes things a bit harder to follow here. I'm adding a comment in the meantime.

Reply:
A comment should probably be fine, but do you really think a method makes it harder to read? Would be that method called something like `StopHandlerAndListeners`? In that case, I'd argue that it is better to read.

Reply:
Alright, done. But the comment is still needed as it explains something else.